_includes/tutorials/aggregating-average/kstreams/markup/dev/run-producer.adoc

```diff
@@ -0,0 +1,23 @@
+////
+   Example content file for how to include a console produer(s) in the tutorial.
+   Usually you'll include a line referencing the script to run the console producer and also include some content
+   describing how to input data as shown below.
+
+   Again modify this file as you need for your tutorial, as this is just sample content.  You also may have more than one
+   console producer to run depending on how you structure your tutorial
+
+////
+
+In a new terminal, run:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/aggregating-average/kstreams/code/tutorial-steps/dev/console-producer.sh %}</code></pre>
++++++
+
+When the console producer starts, it will log some messages and hang, waiting for your input. Copy and paste one line at a time and press enter to send it. Note that these lines contain hard tabs between the key and the value, so retyping them without the tab will not work.
```

Comment:
This paragraph has some out of date info, it's also not really necessary. I suggest deleting it altogether.

```suggestion
```